### PR TITLE
Feature/gh 5121

### LINKED ISSUE #5121: Submit return request
Implement this in OCC, store and facades layer. 

A dummy button was added in #5119, you need to make sure that clicking this button, submit request works. Customer will be redirect to "order return request details" page. The "order return request details" page will be added in #5477. So, this "redirect" will be refactored in #5477.

### PR DESCRIPTION
**Pull request by WeizhengSap**
close #5121

### CODE REVIEW

File: projects/core/src/occ/adapters/user/default-occ-user-config.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const defaultOccUserConfig: OccConfig = {
         addressVerification: 'users/${userId}/addresses/verification',
         consignmentTracking:
           'orders/${orderCode}/consignments/${consignmentCode}/tracking',
+        returnOrder: 'users/${userId}/orders/${orderId}/returnRequest',
```
[the remark this answers is not in the document]
Comment (WeizhengSap) [reply projects/core/src/occ/adapters/user/default-occ-user-config.ts]: yes, the API was changed recently :(

File: projects/core/src/occ/adapters/user/occ-user-order.adapter.ts [hunk 2]
```diff
@@ -147,4 +154,27 @@ export class OccUserOrderAdapter implements UserOrderAdapter {
...
+    });
+
+    returnRequestInput = this.converter.convert(
+      returnRequestInput,
+      ORDER_RETURN_REQUEST_NORMALIZER
+    );
+
+    return this.http
+      .post(url, returnRequestInput, { headers })
+      .pipe(catchError((error: any) => throwError(error)));
```
[the remark this answers is not in the document]
Comment (WeizhengSap) [reply projects/core/src/occ/adapters/user/occ-user-order.adapter.ts]: yes

File: projects/core/src/user/connectors/order/converters.ts [hunk 3]
```diff
@@ -10,3 +13,7 @@ export const ORDER_HISTORY_NORMALIZER = new InjectionToken<
 export const CONSIGNMENT_TRACKING_NORMALIZER = new InjectionToken<
   Converter<any, ConsignmentTracking>
 >('ConsignmentTrackingNormalizer');
+
+export const ORDER_RETURN_REQUEST_NORMALIZER = new InjectionToken<
+  Converter<any, ReturnRequestEntryInputList>
```
[the remark this answers is not in the document]
Comment (WeizhengSap) [reply projects/core/src/user/connectors/order/converters.ts]: ok.

File: projects/core/src/occ/adapters/user/default-occ-user-config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (LTiger14) [reply projects/core/src/occ/adapters/user/default-occ-user-config.ts]: It's more my fault than yours :(

**Pull request merged**